docs/multicluster/user-guide.md

```diff
@@ -562,6 +563,30 @@ Normally, `podCIDRs` should be the value of `kube-controller-manager`'s
 will not be enabled. If you use `kubectl edit` to edit the ConfigMap, then you
 need to restart the `antrea-mc-controller` Pod to load the latest configuration.
 
+## Multi-cluster WireGuard Tunnel
```

Comment:
Probably just name it "WireGuard Encryption".

Do you think we can make it a sub-section of "Multi-cluster Gateway Configuration", as WG is configured on Gateways for tunnel traffic between Gateways?

Reply:
Yes, made the WG  section as a sub-section in the Gateway configuration.